Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -169,13 +289,21 @@ export function findReferringDocumentInfoForErrorReportingSql(referringMeadowlar
  * @param aliasId the alias meadowlarkId for the given document, this may be the same as the meadowlarkId
  * @returns SQL query string to insert into the aliases table
  */
-export function insertAliasSql(documentUuid: DocumentUuid, meadowlarkId: MeadowlarkId, aliasId: MeadowlarkId): string {
-  return format(
+export async function executeInsertAlias(
```

Comment:
Not a strong feeling here, but consider whether the `execute` prefix on these fuinction names are worthwhile.

Reply:
When I click on "Files changed" tab and search for "function executeInsertAlias" I cannot find this function at all. I wanted to review the content of the function before commenting. This seems rather odd to me.